docusaurus/docs/Android/04-compose/04-message-components/02-message-list-header.mdx

```diff
@@ -56,30 +63,24 @@ fun MessageListHeader(
 )
 ```
 
-* `onBackPressed`: Handler for the `BackButton` click action.
+* `onBackPressed`: Handler for the header `BackButton` click action.
```

Comment:
Felt that it was important to disambiguate this because I suspect some users might confuse it with overriding the system back button